pom.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>eu.cloudnetservice</groupId>
+    <artifactId>v2-chatsystem</artifactId>
+    <packaging>pom</packaging>
+    <version>1.0-SNAPSHOT</version>
+
+    <modules>
+        <module>chat-plugin</module>
+    </modules>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+
+        <plugin.shade.version>3.2.1</plugin.shade.version>
+
+        <chat.version>1.0-SNAPSHOT</chat.version>
+
+        <dependency.cloudnet.version>2.1.17</dependency.cloudnet.version>
+        <dependency.spigot.version>1.8.8-R0.1-SNAPSHOT</dependency.spigot.version>
```

Comment:
Given the api-version of this plugin, think about the minimum supported spigot version.

Reply:
Is not necessary, the api-version says only the backward compatibility for spigot 1.13 above

Reply:
I think it's in the spigot.yml, but you can specify that spigot won't load plugins with an API version lower than X.